.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,9 +1,7 @@
 ---
 name: Bug report
-about: Create a report to help us improve
-title: ''
+about: Tell us where it is, we'll squash it
```

Comment:
Same as with performance think we can be a bit more clear on this, a bit like: Something is not working as expected